haas/ext/switches/brocade.py

```diff
@@ -244,7 +244,7 @@ def _disable_native_tag(self, interface):
 
         """
         url = self._construct_url(interface, suffix='trunk/tag/native-vlan')
-        self._make_request('DELETE', url)
+        self._make_request('DELETE', url, log=False)
```

Comment:
Can you check for the excepted error code in the case of the vlan being already disabled and log if it's a different one?